packages/xo-server-sdn-controller/src/ovsdb-client.js

```diff
@@ -35,6 +35,10 @@ export class OvsdbClient {
     return this._host.$id
   }
 
+  get hostMetricsRef() {
+    return this._host.metrics
+  }
```

Comment:
Why not expose the `host` record directly, you're already using it?

Reply:
Seems better indeed, i exposed the field of host to be able to use the `{foo: bar}` syntax in the find but it's making too many getters now...